packages/db/scripts/import-export/tables.txt

```diff
@@ -0,0 +1,38 @@
+users
+apps
+subjects
+key_stages
+lesson_schemas
+chat_user_flags
+aila_user_modifications
+users_allowed_to_view_new_features
+prompts
+app_sessions
+key_stage_subjects
+lessons
+statistics
+transcripts
+questions
+answers
+lesson_summaries
+lesson_plans
+lesson_plan_parts
+downloads
+questions_for_judgement
+answers_and_distractors_for_judgement
+snippets
+comparative_judgements
+comparative_judgement_results
+comparative_judgement_flags
+generations
+user_tweaks
+re_generations
+generation_user_flags
+shared_content
+lesson_snapshots
+lesson_exports
+qd_snapshots
+moderations
+qd_exports
+lesson_export_downloads
+qd_export_downloads
```

Comment:
I did want a more dynamic way of doing this however it was so much more reliable to just have a clear ordered list.